apps/landingPage/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@kiwicom/margarita-landing-page",
+  "private": true,
+  "version": "0.1.0",
+  "dependencies": {
+    "@kiwicom/orbit-components": "^0.28.2",
+    "@kiwicom/orbit-design-tokens": "^0.3.0",
+    "gatsby": "^2.0.105",
+    "gatsby-image": "^2.0.20",
```

Comment:
The `gatsby-image` is nowhere to be used. Or maybe I did not find where?

Reply:
You are right, its also not used